frontend/src/components/nlp/components/NlpTrainForm.tsx

```diff
@@ -121,7 +124,12 @@ const NlpDatasetSample: FC<NlpDatasetSampleProps> = ({
   useQuery({
```

Comment:
```suggestion
  const { isLoading } = useQuery({
```
useQuery() returns isLoading so don't have to add a new state.

Reply:
You're right! I missed that. I'll remove the useState and use the loading state directly from useQuery. Thanks for pointing it out!

Reply:
Hello @marrouchi,

Thank you for your suggestion. I have updated the implementation by removing the useState hook and instead utilizing the loading state from useQuery.